coforma/swift-tech-challenge
# Add dynamo put function and scaffold application utility

# Summary

- Add put function to dynamo lib so we can save records
- Scaffold application utility for saving applications
- Add tests

Fixes #111

## Type of change


# How To Test
- Wait until application submit is ready
- Hook it up and make sure it saves to the db!

# Checklist:
